src/defgen.cpp

```diff
@@ -288,7 +288,7 @@ void generateDEFForMember(MemberDef *md,
     {
       if (rmd->getStartBodyLine()!=-1 && rmd->getBodyDef())
       {
-        t << memPrefix << "referenceby = {" << endl;
+        t << memPrefix << "referencedby = {" << endl;
```

Comment:
This is an output change, but I think this is debug code...

Reply:
"That is an output change", this is the reason why I didn't change it .

As far as I can tell the output is used in case `GENERATE_AUTOGEN_DEF` is set, but I think too that it wouldn't be a problem to change it (but I'm not sure as I don't know autogen, project looks a bit dead to me though).